packages/dashboard/src/components/dashboard/index.test.tsx

```diff
@@ -19,7 +19,7 @@ it('renders', function () {
           numRows: 100,
         },
         widgets: [],
-        viewport: { duration: '5m' },
```

Comment:
instead of changing this test could you create a new test which does the following:

1. check that if no initial viewport is passed in, the dashboard uses a default viewport
2. check that if a viewport is passed in, the dashboard uses that value instead